### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by mondeja**
Changes proposed in this pull request:

* Fix `clamp()` arguments not being correctly rendered with `mkdocstrings`.
* Use [`include-markdown`](https://github.com/mondeja/mkdocs-include-markdown-plugin#mkdocs-include-markdown-plugin) plugin to include "Usage" section of README inside Home.
* Don't allow to select `>>>` text nodes inside codeblocks. For `pycon` code blocks use [the recipe suggested by mkdocstrings](https://mkdocstrings.github.io/recipes/#prevent-selection-of-in-python-code-blocks) ~~and for `python` codeblocks a Javascript function with a CSS class~~.
* Install mkdocstrings with the legacy Python handler as extra (`mkdocstrings[python-legacy]>=0.18`). See [About the Python handlers](https://mkdocstrings.github.io/handlers/overview/#about-the-python-handlers) in mkdocstrings documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,13 @@ human-readable size or throughput. It is localized to:
...
 [https://python-humanize.readthedocs.io](https://python-humanize.readthedocs.io)
 
+<!-- usage-start -->
+
 ## Usage
 
 ### Integer humanization
 
-```pycon
+```python
```
Comment (hugovk) [inline README.md:51]: Let's keep `pycon` for these with Python console output:

image

https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer

File: docs/css/code_select.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* Don't allow to select >>> in ```pycon``` code blocks */
+.highlight .gp, .highlight .go { /* Generic.Prompt, Generic.Output */
+    user-select: none;
+}
+
+/* >>> selection in ```python``` code blocks is handled with Javascript */
```
Comment (hugovk) [inline docs/css/code_select.css:6]:
```suggestion
/* >>> selection in ```python``` code blocks is handled with JavaScript */
```

**Review by hugovk — CHANGES_REQUESTED** [review]
Thank you very much for this, nice improvements! 

---

> * Don't allow to select `>>>` text nodes inside codeblocks. For `pycon` code blocks uses [the recipe suggested by mkdocstrings](https://mkdocstrings.github.io/recipes/#prevent-selection-of-in-python-code-blocks) and for `python` codeblocks a Javascript function with a CSS class.

I'm not sure if this is working?

For example, at `/number/#humanize.number.clamp`, when I click the "Copy to clipboard icon", it copies this:

```
>>> clamp(123.456)
'123.456'
>>> clamp(0.0001, floor=0.01)
'<0.01'
>>> clamp(0.99, format="{:.0%}", ceil=0.99)
'99%'
>>> clamp(0.999, format="{:.0%}", ceil=0.99)
'>99%'
>>> clamp(1, format=intword, floor=1e6, floor_token="under ")
'under 1.0 million'
>>> clamp(None) is None
True
```

Should it instead copy something like this?

```
clamp(123.456)
clamp(0.0001, floor=0.01)
clamp(0.99, format="{:.0%}", ceil=0.99)
clamp(0.999, format="{:.0%}", ceil=0.99)
clamp(1, format=intword, floor=1e6, floor_token="under ")
clamp(None) is None
```

**Comment by pawamoy** [comment]
We should probably use `pycon` for code blocks detected in examples, and provide the CSS rule to prevent selection of `>>>` and `...` by default in mkdocstrings :slightly_smiling_face:

**Comment by mondeja** [comment]
> when I click the "Copy to clipboard icon", it copies this...

Opened an issue in [mkdocs-material#3647](https://github.com/squidfunk/mkdocs-material/issues/3647) and seems that the unique solution would be to hide the "Copy to clipboard" buttons.

I've pinned `pymdown-extensions>=9.2` because the function `pygments_lang_class` has been included in that version. Also updated the CSS selectors to only target `pycon` codeblocks.

**Comment by pawamoy** [comment]
I didn't know about that pygment option, I'm not sure how to include these CSS rules by default in the new Python handler now :confused:

**Comment by mondeja** [comment]
> Maybe we shouldn't include them by default, as pointed out in the issue you opened.

I agree, what do you think @hugovk?

**Comment by hugovk** [comment]
Sure, fine to disable it.

The [Sphinx version of the copy button](https://sphinx-copybutton.readthedocs.io/) can strip out `>>>` but I understand if it's too complex for mkdocs-material to implement. We don't have all that much code examples here.

Thanks!

**Review by hugovk — APPROVED** [review]

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**